baseledger/ops/deployment_and_chain_upgrade.md

```diff
@@ -104,4 +104,38 @@ Open a new terminal and submit an upgrade propoposal
 
     baseledgerd tx gov vote 1 yes --from validator --yes
 
-This will create a proposal which is voted pass. when the current version of the node reaches this height, it will halt consensus on all nodes, create a upgrade-info.json that will be read by cosmosvisor and will trigger a switch of the sym link to upgrade folder. Start of the chain will trigger migrations defined in the new app and the chain will continue consensus. 
\ No newline at end of file
+This will create a proposal which is voted pass. when the current version of the node reaches this height, it will halt consensus on all nodes, create a upgrade-info.json that will be read by cosmosvisor and will trigger a switch of the sym link to upgrade folder. Start of the chain will trigger migrations defined in the new app and the chain will continue consensus. 
+
+
+## Example local test of parameter change proposal
+
+    baseledgerd unsafe-reset-all
+    baseledgerd config chain-id baseledger
+    baseledgerd config keyring-backend test
+    baseledgerd config broadcast-mode block
+    baseledgerd init baseledger --chain-id baseledger --overwrite
+
+Set voting time to 20s in /root/.baseledger/config/genesis.json
+
+    baseledgerd keys add validator
+    baseledgerd add-genesis-account validator 5000000000stake --keyring-backend test
+    baseledgerd gentx validator 1000000stake --chain-id baseledger
+    baseledgerd collect-gentxs
+    baseledgerd start
+
+
+Open a new terminal and submit a param change propoposal
+
+    copy the ./params.json to the folder where you are running the command
+
+    baseledgerd tx gov submit-proposal param-change param.json  --from validator     --yes
+
+    baseledgerd tx gov deposit 1 10000000stake --from validator --yes
```

Comment:
To prevent spamming the network with proposals. I would add proper instructions for governance proposals as part of out testnet setup next week, these here are just for quick tests.